S-L-Murphey/Gamer-Rater-API
# SLM GamerRater

#### Changes Made

1. Added file `models` to `raterapp` directory. Along with all the fields.

#### Steps to Review

1. Checkout to this branch locally
   ```
   git fetch --all
   git checkout slm-gamerrater
   ```
2. Ensure the backend and client are both running.
4. Test app functionality.

   > Example: Pull this project down and check out the models app for correctness.

5. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
